### DISCUSSION TITLE
Calibration window not scaling

### DISCUSSION THREAD

**Discussion by hydrolicus** [sn4k3/UVtools#349] [General]
Unfortunately the calibration window is not scaling so I can use the interface. Any thoughts as to how to fix?

![image](https://user-images.githubusercontent.com/81529167/142659058-2dada3f1-7cfb-41a8-8d28-5d82f397b2a4.png)

![20211119_141521 (1)](https://user-images.githubusercontent.com/81529167/142679077-fe0120df-88ed-4271-ba33-b25d080e9d00.jpg)

**Comment by sn4k3** [comment]
Missing information...
Go to Help - About. Take screenshot of that screen and update OP

**Reply by hydrolicus** [reply]
done I think, Not sure what OP stands for...

**Reply by sn4k3** [reply]
It's the main post.
Why screenshot is cut? I need the whole information, that includes the screens that is cut

**Reply by sn4k3** [reply]
Ok, if that is cut i think your problem is with screen scalling. It looks like you under 150% or something. The UI framework don't like scalling too much. If you set to 100% all your problems will be solved.

Also post a screenshot of the Settings windows on General Tab
